zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/AzureNotebookRepo.java

```diff
@@ -205,8 +205,21 @@ public void close() {
   }
 
   @Override
-  public void checkpoint(String noteId, String checkPointName) throws IOException {
+  public Rev checkpoint(String noteId, String checkpointMsg) throws IOException {
```

Comment:
How do you think, is there a reason why we do not want a readable and concise class name like `Revision` to be used here? Or something like `Checkpoint` as the same concept is referred in API